monitoring/README.md

```diff
@@ -92,47 +157,113 @@ c) Upload the dashboard JSON files from your local `./grafana` sub-directory int
+     default           kubernetes                                                        172.18.0.2:6443                                          3d20h
+     its1-system       vcluster                                                          10.244.0.12:8443,10.244.0.12:8443                        3d20h
+     wds1-system       wds1                                                              10.244.0.21:9444                                         3d20h
+     ```
+     
+     If you're using kind clusters then use the following command to obtain the IP address of the host container for your cluster: `$ docker inspect -f '{{range.NetworkSettings.Networks}}{{.IPAddress}}{{end}}' <docker-container-name>`
+
+     For example: 
+     ```bash
+     docker inspect -f '{{range.NetworkSettings.Networks}}{{.IPAddress}}{{end}}' cluster1-control-plane
+     ```
+
+     Output:
+     ```bash
+       172.18.0.3
+     ```
+
+Here are the default port number for the APIservers: 
+
+| APIServer Pod | Default Port #|
+| ------------- | ------------- |
+| Host-cluster  |     6443      |
+| WDS space     |     6443      |
+| ITS space     |     8443      |
+| WEC-cluster   |     9444      |
 
-Note: you can also monitor the APIserver of the hosting cluster using the following values: `NS: default` and `APIService: apiserver`
 
 b) API Server Priority & Fairness: input into the following text boxes.
 - `NS`: name of the KubeStellar space namespace, e.g., *wds1-system*, *wds2-system*, *its1-system*, etc.
 - `APIService`: name of the service for a KubeStellar space APIserver, e.g., *wds1*, *wds2*, *vcluster*, etc.
+- `Instance`: hostname and port number for the target KubeStellar space APIserver, e.g., *10.244.0.18:9444*
 
     <img src="images/apiserver-apf-monitoring.png" width="60%" height="80%" title="apiserver-apf-monitoring.png"> 
 
-Note: use the same combinations of `NS` and `APIService` parameters described in `step 5-a` to monitor the APIserver in the WDS space, ITS space or the hosting cluster. 
+**Note**: use the same combinations of `NS` and `APIService` parameters described in `step 5-a` to monitor the APIserver in the WDS space, ITS space, the hosting or WEC clusters. 
 
 
 c) KubeStellar Controllers Monitoring: input into the following text box and drop-down menu. 
 
-- `SpaceNS`: name of the KubeStellar space namespace, e.g., *wds1-system*, *wds2-system*, etc.
+- `NS`: name of the KubeStellar space namespace, e.g., *wds1-system*, *wds2-system*, etc.
 
-- `KSController`: select the KubeStellar controller name. Available options from the drop-down menu: *kubestellar-controller-manager-metrics-service*, *ks-transport-controller* and *status-addon-controller*. 
+- `KSController`: select the KubeStellar controller name. Available options from the drop-down menu: *kubestellar-controller-manager-metrics-service*, *ks-transport-controller*, *status-addon-controller* and *status-agent-controller*. 
```

Comment:
"status-agent-controller" is not a thing. The "OCM Status add-on agent" _is_ a thing.

Reply:
My point is not about spaces vs dashes. My point is that the architecture document does not talk about a "status agent controller" (with any punctuation); I think the intended thing is what the architecture document calls the "OCM Status Add-on Agent". I am OK with dropping "OCM" from the name in the menu. But putting "controller" in there is not what we want, I think.

Reply:
I will address this in a new PR. I created an issue to keep track of this here: https://github.com/kubestellar/kubestellar/issues/2470